applications/ConvectionDiffusionApplication/python_scripts/conjugate_heat_transfer_solver.py

```diff
@@ -96,8 +112,11 @@ def _validate_settings(self, custom_settings):
         }
         """)
 
-        ## Overwrite the default settings with user-provided parameters
-        self.settings.ValidateAndAssignDefaults(default_settings)
+        default_settings.AddMissingParameters(super().GetDefaultParameters())
+        return default_settings
+
+    def ValidateSettings(self):
```

Comment:
looks pretty good, except that I would also call the baseclass 

see here how we solved it in the structure (I think is a good solution)

https://github.com/KratosMultiphysics/Kratos/blob/e1f1c94e9195372545873ac4bcb8a6a9e58270b9/applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py#L141-L147

Reply:
Done.